packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: snowplow/snowplow_utils
-    version: [">=0.13.0", "<0.14.0"]
+    version: [">=0.13.1", "<0.14.0"]
```

Comment:
I understood so far that we have this practice to mark this not to forget to add this for the prepare for release commit in the release pr

Reply:
No I don't think so. We usually have a `to delete` commit when we pin it to a temporary branch that needs to be released, but we do the bump in the initial development as well, so this can be in the main commit (unless I'm misremembering)

Reply:
That's fine, another of those conventions that we can clarify for everyone to follow on the monthly catchup. I have changed the commit message to leave this in.